### DISCUSSION TITLE
mysqld Event Scheduler Error

### DISCUSSION THREAD

**Discussion by inthreedee** [photoprism/photoprism#2805] [Q&A] [answered]
I'm testing the preview release in relation to https://github.com/photoprism/photoprism/issues/2726 and see this mysql error in my logs. Is this something to worry about?

```
mariadb_1     | 2022-10-26 18:17:18 0 [ERROR] Incorrect definition of table mysql.event: expected column 'definer' at position 3 to have type varchar(, found type char(141).
mariadb_1     | 2022-10-26 18:17:18 0 [ERROR] mysqld: Event Scheduler: An error occurred when initializing system tables. Disabling the Event Scheduler.
mariadb_1     | 2022-10-26 18:17:18 0 [Note] mysqld: ready for connections.
```

**Comment by lastzero** [comment, marked as answer]
Looks like an internal MariaDB table? Probably requires an update then:

👉 https://docs.photoprism.app/getting-started/troubleshooting/mariadb/#version-upgrade

**Comment by inthreedee** [comment]
Looks like that worked, thanks!
